daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -93,6 +93,28 @@ class RazerMambaTE(__RazerDeviceBrightnessSuspend):
     DPI_MAX = 16000
 
 
+class RazerLanceheadTE(__RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Lancehead Tournament Edition (Wired)
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x0060
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 16]
+    METHODS = ['get_firmware', 'get_matrix_dims', 'has_matrix', 'get_device_name', 'get_device_type_mouse', 'get_brightness', 'set_brightness', 'set_wave_effect',
+               'set_static_effect', 'set_spectrum_effect', 'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect',
+               'set_breath_single_effect', 'set_breath_dual_effect', 'set_custom_effect', 'set_key_row', 'max_dpi',
+               'get_dpi_xy', 'set_dpi_xy']
+
+    RAZER_URLS = {
+        "top_img": "https://d1urewwzb2qwii.cloudfront.net/sys-master/root/h5e/h0b/8888236245022",
```

Comment:
Aren't there images available on the Razer site? I'd rather not have externally hosted images.

Reply:
yeah.. tried to look the generated html page and didn't see any reference. Also I have tried several different naming patterns and didn't find any match.. =/
The product ID is: 27310,27309

For example:
https://assets.razerzone.com/eeimages/products/27310/lanceheadte-og-1200x630-v1.png
https://assets.razerzone.com/eeimages/products/27309/ogimage-1200x630.jpg

Reply:
side_img: https://assets.razerzone.com/eeimages/products/27310/lanceheadte-galleryimg-1500x1000-4.png
perspective_img: https://assets.razerzone.com/eeimages/products/27310/lanceheadte-galleryimg-1500x1000-5.png
Didn't find a nice top_img yet